filament/backend/src/metal/MetalEnums.h

```diff
@@ -296,6 +296,27 @@ constexpr inline MTLTextureType getMetalType(SamplerType target) {
     }
 }
 
+inline MTLTextureType getMetalTypeMS(SamplerType target) {
+    switch (target) {
+        case SamplerType::SAMPLER_2D:
+        case SamplerType::SAMPLER_EXTERNAL:
+            return MTLTextureType2DMultisample;
+        case SamplerType::SAMPLER_2D_ARRAY: {
+            if (@available(iOS 14.0, macCatalyst 14.0, *)) {
+                return MTLTextureType2DMultisampleArray;
+            }
+            else {
+                ASSERT_POSTCONDITION(false, "MTLTextureType2DMultisampleArray not supported on this platform.");
+                return MTLTextureType2D;
+            }
+        }
+        default: {
+            ASSERT_POSTCONDITION(false, "There are no multisample variant of this Metal texture type.");
```

Comment:
nit: 
```suggestion
            ASSERT_POSTCONDITION(false, "There is no multisample variant of this Metal texture type.");
```

Reply:
fixed in https://github.com/shapr3d/filament/pull/67/commits/3f472d0b95e6d2f3283658f4591e8952ff216133